lib/routes/api/users.js

```diff
@@ -179,12 +188,57 @@ router.get('/me', async(req, res) => {
   }
 });
 
+router.get('/:user_sid', async(req, res) => {
+  const logger = req.app.locals.logger;
+  const token = req.user.jwt;
+  const decodedJwt = jwt.verify(token, process.env.JWT_SECRET);
+  const {user_sid} = req.params;
+
+  try {
+    const user = await User.retrieve(user_sid);
+    delete user.hashed_password;
+    // const {hashed_password, ...rest} = user;
```

Comment:
@davehorton I have an issue with removing the hashed_password before sending the result in a response, but `delete user.hashed_password;` seems not to work, I assume because it is a referenced property. I wanted to use ...rest as in the commented line, but this is not allowed by the linter, since later on, I am not using the hashed_password variable.

Reply:
I would use the spread syntax that you have.  just add this
```
    // eslint-disable-next-line no-unused-vars
    const {hashed_password, ...rest} = user;
```